solc/CommandLineInterface.h

```diff
@@ -81,6 +81,12 @@ class CommandLineInterface
 	/// @arg _data to be written
 	void createFile(std::string const& _fileName, std::string const& _data);
 
+	/// Create a json file in the given directory
+	/// @arg _fileName the name of the file (the extension will be replaced with .json)
+	/// @arg _json to be written
+	/// @arg pretty boolean indicating the json to be written in compact or pretty form (true by default)
+	void createJson(std::string const& _fileName, Json::Value const& _json, bool pretty=true);
```

Comment:
Please add spaces around `=`.

This also changes behaviour.

Perhaps it would make more sense adding a flag, `--prettyprint` to output pretty printed JSON?